src/unstable/components/Form/widgets/BaseInput.tsx

```diff
@@ -28,13 +36,18 @@ const BaseInput = (props: FormWidgetProps) => {
 		return props.onChange(value === '' ? options.emptyValue : value);
 	};
 
+	const suggestions = filterSuggestions(value, schema.examples as string[]);
+
 	return (
 		<Input
 			width="100%"
 			readOnly={readonly}
 			disabled={disabled}
 			autoFocus={autofocus}
-			autoComplete={options.autocomplete}
+			// Form suggestions will clash with browser suggestions if autocomplete is on
+			autoComplete={
```

Comment:
autoComplete off doesn't work on multiple versions of chrome. I have found that adding a blank `<datalist />` does disables the browsers autocomplete.

Reply:
👍